src/app/services/configuration.service.ts

```diff
@@ -37,7 +37,7 @@ export class ConfigurationService {
         const categoryName = encodeURIComponent(category_name);
         let body = JSON.stringify({ 'value': value });
         if (type.toUpperCase() === 'JSON') {
-            body = JSON.stringify({ 'value': JSON.parse(value) });
+            body = JSON.stringify({ 'value': value });
```

Comment:
@monika-Dianomic @mshariq-nerd 

Once this is in,
https://github.com/foglamp/FogLAMP/pull/953#discussion_r204904754
1) we shall need `app/components/core/configuration-manager/add-config-item/add-config-item.component.html` to have textarea as well for type JSON.

Reply:
and this will go to 1433-main! not the develop as f953 is for f1432.